.github/workflows/test.yml

```diff
@@ -52,4 +53,7 @@ jobs:
         docker exec CI_container /bin/bash -c 'cd ./Package; \
           source ${{ matrix.STACK }}; \
           source ./setup.sh; \
-          fccanalysis run examples/FCCee/higgs/mH-recoil/mumu/analysis_stage1.py --output myoutput.root --files-list root://eospublic.cern.ch//eos/experiment/fcc/ee/generation/DelphesEvents/spring2021/IDEA/p8_ee_Zbb_ecm91_EvtGen_Bc2TauNuTAUHADNU/events_131527278.root'
+          fccanalysis run examples/FCCee/higgs/mH-recoil/mumu/analysis_stage1.py --output myoutput.root --files-list root://eospublic.cern.ch//eos/experiment/fcc/hh/tutorials/edm4hep_tutorial_data/p8_ee_ZH_ecm240.root'
```

Comment:
instead of `--files-list root://eospublic.cern.ch//eos/experiment/fcc/ee/generation/DelphesEvents/spring2021/IDEA/p8_ee_Zbb_ecm91_EvtGen_Bc2TauNuTAUHADNU/events_131527278.root` let's use `--test`.